cmake/modules/VTA.cmake

```diff
@@ -60,6 +60,11 @@ elseif(PYTHON)
     find_library(__cma_lib NAMES cma PATH /usr/lib)
     target_link_libraries(vta ${__cma_lib})
   endif()
+
+  # DPI module
+  include_directories("vta/include")
+  file(GLOB RUNTIME_DPI_SRCS vta/src/verilator/dpi_module.cc)
```

Comment:
My guess is dpi_module is not specific to verilator. we can put it under vta/src/dpi